Dockerfile

```diff
@@ -1,4 +1,12 @@
-FROM concourse/buildroot:curl
+FROM progrium/busybox
```

Comment:
Probably, yea. Shouldn't need the cert stuff either, the base buildroot image provides it.